skimage/feature/_canny.py

```diff
@@ -67,6 +68,9 @@ def canny(image, sigma=1., low_threshold=None, high_threshold=None, mask=None):
         If None, high_threshold is set to 20% of dtype's max.
     mask : array, dtype=bool, optional
         Mask to limit the application of Canny to a certain area.
+    quantile_threshold : bool, optional
+        If True then treat low_threshold and high_threshold as quantiles of the
+        edge magnitude image, rather than absolute edge magnitude values.
```

Comment:
Mention that thresholds should be in the range [0, 1] if True, since speaking of "quantiles" might be confusing.